src/main/java/frc/robot/RobotContainer.java

```diff
@@ -6,13 +6,39 @@
 
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.commands.ShooterBaseCommand;
+import frc.robot.subsystems.shooter.ShooterBase;
+import frc.robot.subsystems.shooter.ShooterIOSim;
 
 public class RobotContainer {
+  public final CommandXboxController controller;
+  public final ShooterBase shooter;
+
   public RobotContainer() {
+    controller = new CommandXboxController(0);
+
+    boolean swerveBot = false;
+    boolean tankBot = false;
+
+    if (swerveBot) {
+      shooter = null;
+    } else if (tankBot) {
+      shooter = null;
+    } else {
+      shooter = new ShooterBase(new ShooterIOSim());
+    }
+
+    shooter.setDefaultCommand(new InstantCommand(() -> shooter.disable(), shooter));
+
     configureBindings();
   }
 
-  private void configureBindings() {}
+  private void configureBindings() {
+    controller.rightTrigger().onTrue(new ShooterBaseCommand(shooter, () -> true));
+    controller.rightTrigger().onFalse(shooter.getDefaultCommand());
```

Comment:
This works, but a simpler solution would be to replace these two lines with a single line using [**while**True](https://docs.wpilib.org/en/stable/docs/software/commandbased/binding-commands-to-triggers.html#whiletrue):

`controller.rightTrigger().whileTrue(new ShooterBaseCommand(shooter, () -> true));`

The command is cancelled when the button is released, and if nothing else is scheduled for the shooter, it'll _automatically_ go back to the default command.

Reply:
Just did it and it still works.